metaworld/policies/__init__.py

```diff
@@ -0,0 +1,7 @@
+from metaworld.policies.sawyer_window_open_policy import SawyerWindowOpenPolicy
+from metaworld.policies.sawyer_window_close_policy import SawyerWindowClosePolicy
+
+__all__ = [
+    'SawyerWindowOpenPolicy',
```

Comment:
please name these according to the canonical names (e.g. door-close-v1, etc.) and always include the version.

Reply:
These policies work for both v1 and v2. Should I make duplicate classes with that naming scheme?